README.md

```diff
@@ -7,6 +7,9 @@ These findings are based on reverse engineering of the NVIDIA GPU Linux driver.
 ## Prerequisites
 
 - Kernel boot parameter: iomem=relaxed
+
+This step is optional. On some versions, the default kernel boot parameter is sufficient to run this program.
```

Comment:
This step is optional. In some Linux distributions, the default kernel boot parameters are sufficient to run this program.

Reply:
done